github/github.go

```diff
@@ -147,6 +147,9 @@ const (
 
 	// https://developer.github.com/changes/2019-04-11-pulls-branches-for-commit/
 	mediaTypeListPullsOrBranchesForCommitPreview = "application/vnd.github.groot-preview+json"
+
+	// https://developer.github.com/changes/2019-06-12-team-sync/
+	mediaTypeTeamSync = "application/vnd.github.team-sync-preview+json"
```

Comment:
Can you please add `Preview` to the name to make it `mediaTypeTeamSyncPreview` ?